pytorch/src/main/resources/org/bytedeco/pytorch/presets/torch_parselist

```diff
@@ -0,0 +1,1415 @@
+torch/nn/init.h
+torch/nn/modules/common.h
+torch/nn/modules/container/any.h
+#torch/nn/modules/container/functional.h # Complex variadic templates non parseable
+torch/nn/modules/container/moduledict.h
+torch/nn/modules/container/modulelist.h
+torch/nn/modules/container/named_any.h
+torch/nn/modules/container/parameterdict.h
+torch/nn/modules/container/parameterlist.h
+torch/nn/modules/container/sequential.h
+torch/nn/modules/linear.h
+torch/nn/modules/activation.h
+torch/nn/options/adaptive.h
+torch/nn/modules/adaptive.h
+torch/nn/modules/batchnorm.h
+#c10/util/overloaded.h # Non parseable
+torch/nn/modules/conv.h
+torch/nn/modules/distance.h
+torch/nn/modules/dropout.h
+torch/nn/modules/embedding.h
+torch/nn/modules/fold.h
+torch/nn/modules/instancenorm.h
+torch/nn/modules/loss.h
+torch/nn/modules/_functions.h
+torch/nn/modules/normalization.h
+torch/nn/modules/padding.h
+torch/nn/modules/pixelshuffle.h
+torch/nn/modules/pooling.h
+torch/nn/options/rnn.h
+torch/nn/utils/rnn.h
+torch/nn/modules/rnn.h
+torch/nn/options/transformerlayer.h
+torch/nn/options/transformer.h
+torch/nn/modules/transformer.h
+torch/nn/modules/transformerlayer.h
+torch/nn/options/transformercoder.h
+torch/nn/modules/transformercoder.h
+torch/nn/modules/upsampling.h
+torch/nn/modules.h
+torch/nn/options.h
+torch/nn/utils/clip_grad.h
+torch/nn/utils/convert_parameters.h
+torch/nn/utils.h
+torch/nn.h
+torch/optim/optimizer.h
+torch/optim/serialize.h
+torch/optim/adagrad.h
+torch/optim/adam.h
+torch/optim/adamw.h
+torch/optim/lbfgs.h
+torch/optim/rmsprop.h
+torch/optim/sgd.h
+torch/optim/schedulers/lr_scheduler.h
+torch/optim/schedulers/step_lr.h
+torch/optim.h
+torch/sparse.h
+torch/special.h
+torch/version.h
+torch/csrc/api/include/torch/all.h
+
+# Included by
+# ATen/native/TensorShape.h
+# torch/csrc/jit/serialization/storage_context.h
+# torch/csrc/jit/serialization/import.h
+caffe2/serialize/inline_container.h
+caffe2/serialize/istream_adapter.h
+caffe2/serialize/read_adapter_interface.h
+caffe2/serialize/versions.h
+torch/csrc/jit/serialization/unpickler.h
+torch/csrc/jit/frontend/script_type_parser.h
+torch/csrc/jit/frontend/resolver.h
+torch/csrc/jit/frontend/sugared_value.h
+torch/csrc/jit/frontend/error_report.h
+torch/csrc/jit/frontend/tree.h
+torch/csrc/jit/frontend/lexer.h
+torch/csrc/jit/frontend/parser_constants.h
+torch/csrc/jit/frontend/strtod.h
+torch/csrc/jit/frontend/schema_matching.h
+torch/csrc/jit/frontend/versioned_symbols.h
+torch/csrc/jit/frontend/tree_views.h
+torch/csrc/jit/serialization/pickler.h
+
+# Parsed and for inclusion in JNI
+# See also https://github.com/pytorch/pytorch/blob/main/docs/cpp/source/Doxyfile
+# for an approximation of what should be in API in addition to torch.h
+# torch/csrc/jit/runtime/custom_operator.h:  Name conflict with torch::RegisterOperator + little chance to have any use
+torch/torch.h
+ATen/native/TensorShape.h
+torch/csrc/jit/serialization/storage_context.h
+torch/csrc/jit/serialization/import.h
```

Comment:
Instead of trying to come up with some sort of new propriety format just for fun, please use an existing format. I'd go with a subset of C++ that can be interpreted as a normal header like `torch_include.h` like this:
```cpp
// All files included by #include <torch/torch.h>
// as listed by g++ -M torch/csrc/api/include/torch/all.h (or -H)
// Excluding:
// - the ones that fill at::meta at::native and at::_ops namespaces
//   (ATen/ops/*_native.h ATen/ops/*_meta.h ATen/ops/*_ops.h)
// - ATen/ops/_*
// - and some exceptions commented below
#include "torch/csrc/utils/python_stub.h"
#include "c10/macros/cmake_macros.h"
#include "c10/macros/Export.h"
...
```

Reply:
I used what was the easiest to parse while allowing comments.
So that we introduce the minimum and simplest code to maintain, which usually is the way you prefer.
Why not a .h style format... Is there an easy way to bulk-insert in into the JNI without parsing ?
Else I guess I can parse that with a regexp without much more line of codes.

Reply:
Forget what I said about bulk-inserting. We are talking about the list of headers to parse, not to include in JNI.

Reply:
Done. It's better like this indeed.